backend/src/packages/auth/auth.service.ts

```diff
@@ -18,6 +25,33 @@ class AuthService {
     return this.userService.create(userRequestDto);
   }
 
+  public async verifyLoginCredentials({
+    email,
+    password,
+  }: UserSignInRequestDto): Promise<UserSignInResponseDto | null> {
+    const user = await this.userService.findByEmail(email);
+
+    if (!user) {
+      throw new UserNotFoundError({
+        message: 'User with these credentials was not found',
+      });
+    }
+
+    const pswrdCompare = await encrypt.compare(password, user.passwordHash);
```

Comment:
By quality criteria we should not use abbreviations
```suggestion
    const isSamePassword = await encrypt.compare(password, user.passwordHash);
```

Reply:
I thought it was a common name, as in B16 examples. I changed it!